pylzy/lzy/env/provisioning/score.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from typing import Callable
+
+from lzy.types import VmResources
+
+ScoreFunctionType = Callable[[VmResources, VmResources], float]
+
+DEFAULT_WEIGHTS = {
```

Comment:
what's the source of this constants? may be we should modify this constants to make correlation with cloud costs to corresponding billing units? not real-time, of course, but just as a starting point

Reply:
Well, this code is already in master, here is just copy-paste from `lzy/api/v1/provisioning.py`
So this PR is wrong place to discuss it.